NitroxModel/Tcp/Connection.cs

```diff
@@ -65,7 +72,7 @@ public void SendPacket(Packet packet, AsyncCallback callback)
                 {
                     Socket.BeginSend(packetData, 0, packetData.Length, 0, callback, Socket);
                 }
-                catch (SocketException se)
+                catch (SocketException)
```

Comment:
I don't really see why. These should only occur when there are read/write errors with the Socket. The connection is marked for closure when they occur. There's no real need to see the exception details IMO.